app/_src/gateway-operator/get-started/konnect/install.md

```diff
@@ -5,8 +5,9 @@ book: kgo-konnect-get-started
 chapter: 1
 ---
 
-{{ site.kgo_product_name }} can deploy and manage data planes attached to a {{ site.konnect_short_name }} control plane. All the services, routes, and plugins are configured in {{ site.konnect_short_name }} and sent to the data planes automatically.
-
-{% include snippets/gateway-operator/install_with_kubectl.md %}
+{{ site.kgo_product_name }} can deploy and manage data planes attached to a {{ site.konnect_short_name }} control plane.
+All the services, routes, and plugins are configured in {{ site.konnect_short_name }} and sent to the data planes automatically.
 
 Once the `gateway-operator-controller-manager` deployment is ready, you can deploy a `DataPlane` resource that is attached to a {{ site.konnect_short_name }} control plane.
+
+{% include md/kgo/prerequisites-kic.md disable_accordian=true version=page.version release=page.release %}
```

Comment:
Should this just be `prerequisites.md` if there's nothing specific to KIC?

Reply:
Makes sense, let me change that.